internal/provider/private_endpoint_connection_resource_test.go

```diff
@@ -238,7 +231,7 @@ resource "cockroach_private_endpoint_services" "services" {
 
 resource "cockroach_private_endpoint_connection" "connection" {
     cluster_id = cockroach_cluster.dedicated.id
-    endpoint_id = "endpoint-id"
+    endpoint_id = "vpce-0011573c1fa6afa3d"
```

Comment:
rather than hard coding this id everywhere, is it possible to use a datasource to fetch the endpoint using something else like name?

Reply:
>  is it possible to use a datasource to fetch the endpoint using something else like name?

To fetch from AWS? We'd need AWS creds and we'd need to figure out a way to share the AWS API across CC eng. Also, the endpoint is only uniquely identified by ID, so we'd still need to store the endpointId somewhere.

I can use a const if that makes it cleaner.